cdap-examples/DataCleansing/src/test/java/co/cask/cdap/examples/datacleansing/SimpleSchemaMatcherTest.java

```diff
@@ -0,0 +1,89 @@
+/*
+ * Copyright © 2015 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package co.cask.cdap.examples.datacleansing;
+
+import co.cask.cdap.api.data.schema.Schema;
+import com.google.gson.JsonObject;
+import org.junit.Assert;
+import org.junit.Test;
+
+public class SimpleSchemaMatcherTest {
+    Schema simplePersonSchema = Schema.recordOf("person",
```

Comment:
I'm considering removing the schema from the mapreduce, so that user can pass their own via runtime arguments, so I will leave this here until that's finalized.